integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,52 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"clickupTeam"'
+          properties:
+            url: ("https://app.clickup.com/" + (.id|tostring))
+            members: ".members[].user.email"
```

Comment:
This will not yield an array [] but a list of emails delimeted by comma. Use '[.members[].user.email]' instead